packages/core/src/common/classes.ts

```diff
@@ -167,6 +167,15 @@ export const OVERLAY_INLINE = `${OVERLAY}-inline`;
 export const OVERLAY_OPEN = `${OVERLAY}-open`;
 export const OVERLAY_SCROLL_CONTAINER = `${OVERLAY}-scroll-container`;
 
+export const PANELSTACK = `${NS}-panel-stack`;
+export const PANELSTACK_CHEVRON = `${PANELSTACK}-chevron`;
+export const PANELSTACK_HEADER = `${PANELSTACK}-header`;
+export const PANELSTACK_HEADER_BACK = `${PANELSTACK}-header-back`;
+export const PANELSTACK_HEADER_SPACER = `${PANELSTACK}-header-spacer`;
+export const PANELSTACK_HEADER_TITLE = `${PANELSTACK}-header-title`;
+export const PANELSTACK_TRANSITION = `${PANELSTACK}-transition`;
+export const PANELSTACK_VIEW = `${PANELSTACK}-view`;
```

Comment:
my gut says that roughly half of these classes should not be necessary through clever use of flex, but i'll have to dig into the code to figure out which ones.